rust/src/container.rs

```diff
@@ -1,20 +1,33 @@
-//! CLI exposing `ostree-rs-ext container`
+//! Exposing `ostree-rs-ext container`
```

Comment:
This is all fine as is but my original thought here was that we'd create a new file...something like `rust/src/sysroot_upgrade.rs` or something - basically Rust code corresponding to `src/daemon/rpmostree-sysroot-upgrader.cxx`.  But...eh.